bin/juxtaposer/tester/db/mssql/mssql.go

```diff
@@ -0,0 +1,107 @@
+package mssql
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/url"
+
+	_ "github.com/denisenkom/go-mssqldb"
+
+	"github.com/cyberark/secretless-broker/bin/juxtaposer/tester/api"
+	sql_db_tester "github.com/cyberark/secretless-broker/bin/juxtaposer/tester/db/sql"
+)
+
+type MssqlTester struct {
+	sql_db_tester.SqlDatabaseTester
+}
+
+func NewTester() (api.DbTester, error) {
```

Comment:
I think golint is going to complain about the exported functions/methods not having comments.